.devcontainer/devcontainer.json

```
@@ -19,7 +19,8 @@
         "dbaeumer.vscode-eslint",
         "esbenp.prettier-vscode",
         "orta.vscode-jest",
-        "EditorConfig.editorconfig"
+        "EditorConfig.editorconfig",
```

```suggestion
        "editorConfig.editorconfig",
```
Should be lowercase based on the discussions in the other thread